src/gam/__init__.py

```diff
@@ -8493,6 +8511,12 @@ def doCreateResoldCustomer():
 
 
 def _getValueFromOAuth(field, credentials=None):
+    if GC_Values[GC_ENABLE_DASA]:
+        if field == 'email':
+            return GC_Values[GC_DASA_ADMIN]
```

Comment:
OK, I see now. Let's not do this. It won't work in most cases anyway with DASA authentication alone. For those commands that end up calling _getValueFromOAuth, we may need to do something else to work with the DASA credentials or, in the case of something like Gmail commands where a service account just won't be able to use Gmail API, we should just fail with an error that the command doesn't support DASA.